src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlObjectSerializerReadContextComplexJson.cs

```diff
@@ -249,7 +249,7 @@ internal void CheckIfTypeNeedsVerifcation(DataContract declaredContract, DataCon
 
         internal void VerifyType(DataContract dataContract)
         {
-            DataContract knownContract = ResolveDataContractFromKnownTypes(dataContract.StableName.Name, dataContract.StableName.Namespace, null /*memberTypeContract*/);
+            DataContract knownContract = ResolveDataContractFromKnownTypes(dataContract.StableName.Name, dataContract.StableName.Namespace, null /*memberTypeContract*/, null);
             if (knownContract == null || knownContract.UnderlyingType != dataContract.UnderlyingType)
```

Comment:
This method can also be removed as it's only called by CheckIfTypeNeedsVerifcation which is dead code. It doesn't exist on the .NET Framework.